tests/test_recovery.py

```diff
@@ -5,8 +5,7 @@
 import shakedown
```

Comment:
when do these tests get run?

Reply:
This test takes > 20 min. to run. I will setup a nightly test that runs it (similar to the `dcos-commons` nightly tests).

Reply:
great!